src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -96,58 +96,62 @@ typedef Particles<
     >
 > PIC_Photons;
 
-/*--------------------------- electrons --------------------------------------*/
+/*--------------------------- ions -------------------------------------------*/
 
 /* ratio relative to BASE_CHARGE and BASE_MASS */
-value_identifier(float_X, MassRatioElectrons, 1.0);
-value_identifier(float_X, ChargeRatioElectrons, 1.0);
+value_identifier(float_X, MassRatioIons, 1836.152672);
+value_identifier(float_X, ChargeRatioIons, -1.0);
 
 typedef bmpl::vector<
     particlePusher<UsedParticlePusher>,
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
     current<UsedParticleCurrentSolver>,
-    massRatio<MassRatioElectrons>,
-    chargeRatio<ChargeRatioElectrons>
-#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
-    , synchrotronPhotons<PIC_Photons>
-#endif
-> ParticleFlagsElectrons;
+    massRatio<MassRatioIons>,
+    chargeRatio<ChargeRatioIons>
+> ParticleFlagsIons;
 
-/*define specie electrons*/
+/*define specie ions*/
 typedef Particles<
     ParticleDescription<
-        bmpl::string<'e'>,
+        bmpl::string<'i'>,
         SuperCellSize,
         DefaultAttributesSeq,
-        ParticleFlagsElectrons
+        ParticleFlagsIons
     >
-> PIC_Electrons;
+> PIC_Ions;
 
-/*--------------------------- ions -------------------------------------------*/
+/*--------------------------- electrons --------------------------------------*/
 
 /* ratio relative to BASE_CHARGE and BASE_MASS */
-value_identifier(float_X, MassRatioIons, 1836.152672);
-value_identifier(float_X, ChargeRatioIons, -1.0);
+value_identifier(float_X, MassRatioElectrons, 1.0);
+value_identifier(float_X, ChargeRatioElectrons, 1.0);
 
 typedef bmpl::vector<
     particlePusher<UsedParticlePusher>,
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
     current<UsedParticleCurrentSolver>,
-    massRatio<MassRatioIons>,
-    chargeRatio<ChargeRatioIons>
-> ParticleFlagsIons;
+    massRatio<MassRatioElectrons>,
+    chargeRatio<ChargeRatioElectrons>
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
+    ,synchrotronPhotons<PIC_Photons>
+#endif
+#if(ENABLE_BREMSSTRAHLUNG == 1)
+    ,bremsstrahlungIons<PIC_Ions>,
```

Comment:
Maybe it is a little bit more complicated and need some small changes in PIConGPU:

```C++
typedef  bmpl::string<'i'> Ion;
typedef  bmpl::string<'e'> Electron;

typdef Particles<
  ParticleDescription<
      Ion,
      SuperCellSize,
     ATTRIBUTELIST,
     bmpl::vector<Ionisation<Electron> >
   >
> PIC_Ion;

typdef Particles<
  ParticleDescription<
      Electron,
      SuperCellSize,
     ATTRIBUTELIST,
     bmpl::vector<Bremsstrahlung<Ion> >
   >
> PIC_Electron;

//...

typedef bmpl::vector<PIC_Electrons,PIC_Ions> VectorAllSpecies;
```

```C++
//SOLVER
template<typename T_Species>
struct Bremsstrahlung
{
  typedef GetSpecies<T_Species, VectorAllSpecies>::type MySpeciesType;
};
```